### PR TITLE
LPS-163808 [Raylife D2C] Null status bug fix

### PR DESCRIPTION
**Pull request by eliasantosf**
Task: [LPS-163808](https://issues.liferay.com/browse/LPS-163808)

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/hooks/useFormActionsDefault.js [hunk 1]
```diff
@@ -93,6 +96,13 @@ const useFormActions = ({
 
 			setError('continueButton', {});
 
+			if (selectedStep.index === CONTACT_INFORMATION_STEP) {
+				status = APPLICATION_STATUS.OPEN;
+			}
+			else {
+				status = APPLICATION_STATUS.INCOMPLETE;
+			}
+
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/hooks/useFormActionsDefault.js:105]: Please, change this. Does it make sense for you?

```suggestion
                        status = APPLICATION_STATUS.OPEN;

			if (selectedStep.index !== CONTACT_INFORMATION_STEP) {
				status = APPLICATION_STATUS.INCOMPLETE;INCOMPLETE
			}

```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added a suggestion.

File: modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/hooks/useFormActionsDefault.js (hunk 1 above)
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/hooks/useFormActionsDefault.js]: Done!

**Review by joanagrego — APPROVED** [review]
Good job! :white_check_mark:

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Comment by eliasantosf** [comment]
ci:forward

**Pull request closed without merging**